Marlin/clean_nozzle.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Marlin 3D Printer Firmware
+ * Copyright (C) 2016 MarlinFirmware [https://github.com/MarlinFirmware/Marlin]
+ *
+ * Based on Sprinter and grbl.
+ * Copyright (C) 2011 Camiel Gubbels / Erik van der Zalm
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef __CLEAN_NOZZLE_H__
+#define __CLEAN_NOZZLE_H__
+
+#include "point_t.h"
+
+extern void do_blocking_move_to(float x, float y, float z);
```

Comment:
I feel like we're getting ahead of ourselves making every new thing into a class (especially non-static classes that pass a `this` to every method call). Could we make an effort from now on to make all the data and methods `static` from now on? Otherwise, I will need to scrub all the classes and do it myself, which is tedious.

If you want to access a function in `Marlin_main.cpp`, you will need to move its declaration to `Marlin.h`. If the compiler was inlining it, then it won't do that anymore.